services/ui-src/src/utils/forms/forms.ts

```diff
@@ -42,6 +42,7 @@ export const formFieldFactory = (
       key: field.id,
       name: field.id,
       hydrate: field.props?.hydrate,
+      fieldType: field.type,
```

Comment:
Error: Passing `fieldType` as an element attribute with `React.createElement()` won't work because of this error. If we need to access a field's type, we have to name it something else or set it directly in the component and pass it through that way. However, I think we have better options.
![Screenshot 2023-01-30 at 17 39 05](https://user-images.githubusercontent.com/27705928/215621498-5ecb4c25-5ff6-4726-a57d-5e6a509f041f.png)